### PR TITLE
Fix access-info for wildcard domains

### PR DESCRIPTION
**Pull request by yussufsh**
Signed-off-by: Yussuf <yussuf@us.ibm.com>

### CODE REVIEW

File: openshift-install-powervs [hunk 1]
```diff
@@ -654,7 +654,9 @@ function cluster_access_info {
     echo "To access the cluster on local system when using 'oc' run: 'export KUBECONFIG=$PWD/kubeconfig'"
     echo "Access the OpenShift web-console here: $($TF output web_console_url)"
     echo "Login to the console with user: \"kubeadmin\", and password: \"$($($TF output bastion_ssh_command | sed 's/,.*//') -q -o StrictHostKeyChecking=no cat /root/openstack-upi/auth/kubeadmin-password)\""
-    [[ $($TF output etc_hosts_entries) ]] && echo "Add the line on local system 'hosts' file: $($TF output etc_hosts_entries)"
+    if [[ $($TF output etc_hosts_entriess 2>/dev/null) ]]; then
```
Comment (bpradipt) [inline openshift-install-powervs:657]: typo? `entriees`

File: openshift-install-powervs [hunk 2]
```diff
@@ -665,7 +667,9 @@ function cluster_access_info {
 #-------------------------------------------------------------------------
 function access-info {
   [[ -d $ARTIFACTS_DIR ]] && cd ./"$ARTIFACTS_DIR" && TF="../$TF"
-  ! cluster_access_info && error "Cluster is not installed"
+  if ! cluster_access_info; then
+    error "Cluster is not installled"
```
Comment (bpradipt) [inline openshift-install-powervs:671]: typo: `installled`

File: openshift-install-powervs (hunk 1 above)
Comment (yussufsh) [reply openshift-install-powervs:657]: hawk eyes!

**Review by bpradipt — APPROVED** [review]
/lgtm

**Pull request merged**